tools/scripts-core/utils/spawn.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-console */
+import { spawn } from 'node:child_process';
+
+export async function mySpawn(cmd, args, opts = {}) {
```

Comment:
what's the benefits than cross-spawn?

Reply:
Not sure at 100% but I think it's because cross-spawn is not an async function.
This is the case with mySpawn.

Reply:
I have removed cross spawn for two reason: 
First I add an issue to get back the exit code properly with it.
Second because there is no release on this package since 3 years